static/templates/actions_popover_content.hbs

```diff
@@ -120,7 +120,4 @@
         </a>
     </li>
 
```

Comment:
This blank line should be removed as well, to preserve the file's general whitespace pattern.